documents/postman/README.md

```diff
@@ -0,0 +1,59 @@
+# Postman
+
+To download Postman, go [here](https://www.postman.com/downloads/). Postman allows you to send requests.
+
+## Postman collection
+
+The intention of this collection is to provide quick, easy functionality to send email, sms, and push notifications.  This collection can be used by business line users integrating with VA Notify to get familiar with the API and send test notifications.  
+
+The postman scripts use the environment variables and populate or update them as the scripts are executed.
+
+Creation, viewing and editing of templates can be done in the Self Service Portal.
+## basic environment variables
+
+These environment variables should be defined before you can execute any of the scripts
+- notification-api-url: `{environment}-api.va.gov/vanotify`
+- notification-api-url-private: `https://{environment}.api.notifications.va.gov
+`
```

Comment:
@edDocMe360 ^ did this ` accidentally get pushed to a new row?

Reply:
just an errant backtick I missed.